documentation/Functions.md

```diff
@@ -141,14 +146,14 @@ Learn more about the workspace directory and working directory on [Structure](ht
         
 ### changeWorkspace <a name="changeWorkspace"></a>
 This function is used to change the workspace of the current tutorial.
-#### parameter
+#### Parameter:
 This function has only one parameter
 1. First parameter:
     - **Required**
     - **Type**- String
     - **Description**- path to a new workspace (relative to working directory)
 #### example 
```

Comment:
"Example" with capitalized initial letters as in the rest of the document. Make all headings throughout the document begin with capital letters

Reply:
Changed.